website/docs/getting-started.mdx

```diff
@@ -22,7 +22,8 @@ but note ⚠️ you may need to alter commands where we are committing files to
 To follow along, you will need the following:
 - A Kubernetes cluster - such as [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
 - A [GitHub](https://github.com) account and [personal access token with repo permissions](https://help.github.com/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
-- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
+- Optionally - the [gitops CLI](./installation.mdx#gitops-cli) to generate a bcrypt hash of a password for logging into the dashboard.
```

Comment:
This isn't quite accurate for this document any more - the guide does tell you to install the CLI, and it uses `create dashboard` instead of `bcrypt-hash`

Reply:
I took inspiration from the wording from the Enterprise docs - https://docs.gitops.weave.works/docs/next/installation/#6-configure-password
> There are several different ways to generate a bcrypt hash, this guide uses gitops get bcrypt-hash from our CLI.

What would you suggest as the wording? I'm happy to update both occurances

Reply:
Ah and guess that would just be for the `main` docs rather than `0.9.3`

Reply:
Yeah, I meant that (only in main) it isn't a prerequisite any more, just like the flux cli isn't a prerequisite - in both cases, we tell you to install them halfway through.

When we put the CLI installation step into place, I was too lazy to add a "if you don't want to use brew, then there's other methods" blurb (read: had nowhere to link to). Now that there's somewhere to link the user (i.e. the install page), we could have the same `For other installation methods, see [the full installation instructions](install.mdx#gitops-cli)` blurb that the flux step has.

Reply:
Ah gotcha, shamefully I had completely missed the step where it installs the gitops CLI on `main`. Will address and add the blurb as described 👍 thank you